src/server/location/location.service.ts

```diff
@@ -140,4 +145,25 @@ export class LocationService {
         .filter((title) => !!title),
     }));
   }
+
+  /**
+   * Resolves with a list of rooms along with their associated campus and
+   * building information. This is used to populate the Room Admin table.
+   */
+  public async getAdminRooms(): Promise<RoomAdminResponse[]> {
```

Comment:
What do you think to renaming this to `getRooms` or similar? I feel like `getAdminRooms` implies (to the consumer of the method) that the method has some connection with the privilege level of the user (which isn't really the case). This naming convention also has an implicit connection to the front-end table consuming the data, which isn't quite what we want to be doing. 

For example, if we decided to call this method elsewhere (for example in a CLI script or an ETL script) - `getAdminRooms` makes a little less sense in that context.